pkg/daemon/daemon.go

```diff
@@ -353,6 +353,18 @@ func NewUnikDaemon(config config.DaemonConfig) (*UnikDaemon, error) {
 		compilers: _compilers,
 	}
 
+	// osv dynamic
+	_compilers[compilers.OSV_DYNAMIC_QEMU] = &osv.OsvQemuCompiler{
```

Comment:
I suggest to do `osvJavaQemuCompiler := &osv.OsvQemuCompiler...` and then use it in both cases. Similar to `osvJavaXenCompiler` in [L330](https://github.com/emc-advanced-dev/unik/pull/99/files#diff-06b532a2a5a5d3ad1d5341b3e00945efR330).